measures/BuildExistingModel/measure.rb

```diff
@@ -252,7 +237,6 @@ def run(model, runner, user_arguments)
 
     if args['os_hescore_directory'].is_initialized
       os_hescore_directory = args['os_hescore_directory'].get
-      hes_hpxml_measures_dir = File.join(os_hescore_directory, 'hpxml-measures')
```

Comment:
This is removed because `HPXMLtoOpenStudio` (not called from the measure anymore) comes _after_ `HPXMLtoHEScore` and `HEScoreRuleset`.